src/package_io/parse_input.star

```diff
@@ -179,3 +189,23 @@ def default_participant():
 			"validator_extra_params": [],
 			"builder_network_params": None
 	}
+
+
+def enrich_mev_extra_params(parsed_arguments_dict, mev_prefix, mev_port):
+	for index, participant in enumerate(parsed_arguments_dict["participants"]):
```

Comment:
I wonder if this could be extracted into a more readable form (map of client type and flags